search/tutorials/tutoria_attribute_configuration.md

```diff
@@ -0,0 +1,140 @@
+# **Attribute Configuration Tutorial**
+
+## Let's get started
+
+Retail Search allows you to configure the product attributes for the search purpose. 
+
+If you want the Retail Search to be able to filter and facet your catalog using this product attribute, you should set it as indexable.
+
+If you want to enable the search by some product attribute, you should make this attribute searchable. Note, that onsy text attributes can be searchable.
+
+In this tutorial you will learn how to configure the product attributes to allow the Retail Search to perform products indexing and search.
```

Comment:
replace "products indexing" with "product indexing".
Singular form is better to use in this case.

Reply:
allow the Retail Search to perform products indexing and search -> I don't think this is true. What you do is you tell Retail Search that particular attribute is searchable or indexable. Product indexing is a different thing.

Reply:
the statement is corrected